index.html

```diff
@@ -8,49 +9,254 @@
+        </a>
+
         <nav class="burger">
             <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
         </nav>
     </header>
-        <section class="content">
-            <div class="introduction">
-                <h1 class="intro">I’m Vishal. <br> Glad to see <br> you!</h1>
-                <p class="description">I’m a software developer! I can help you build a product , feature or website Look
-                    through some of my work and experience! If you like what you see and have a project you need coded,
-                    don’t hestiate tocontact me.</p>
+    <section class="content">
+        <div class="introduction">
+            <h1 class="intro">I’m Vishal. <br> Glad to see <br> you!</h1>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate tocontact me.</p>
+        </div>
+
+        <div class="social-media">
+            <a href="#" class="contact">LET'S CONNECT </a>
+            <ul class="social-link">
+                <li class="tweet"><a href="#">
+                        <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-github fa-2x" aria-hidden="true"></i>
+                    </a>
+                <li><a href="#">
+                        <i class="fa fa-angellist fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+
+    </section>
+
+    <section class="Projects">
+        <div class="card">
+
+            <div class="right">
+                <img class="snapshot" src="images/S0.png" alt="snapshot">
+                <h2>Tonic</h2>
+                <ul class="detail">
+                    <li class="main">CANOPY</li>
+                    <li class="main2">Back End Dev </li>
+                    <li class="main2">2015</li>
+                </ul>
+
+                <div class="paragraph">
+                    <p class="short.descr">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.
+                    </p>
+                </div>
+
+                <ul class="language">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <div class="button"><a href="#" class="see-btndark">See Project</a>
+                </div>
```

Comment:
- [ ] Please I suggest changing this `div` tag to the `button` tag. 😃 
- [ ] Please these changes should be implemented on lines `108`, `134` and `164` in the `index.html` respectively. 😄